src/app/encoder.cpp

```diff
@@ -174,35 +173,30 @@ uint16_t encodeReadOnOffCommand(uint8_t * buffer, uint16_t buf_length, uint8_t d
     return result;
 }
 
-uint16_t _encodeIdentifyClusterCommand(uint8_t * buffer, uint16_t buf_length, uint8_t command_id, uint8_t destination_endpoint,
-                                       uint16_t * identify_duration)
-{
+/*
+ * Identify Cluster commands
+ *
+ * Pick cluster id as 0x0003 for now
```

Comment:
sould 0x0003 have a constant instead?

it seems to show up in comments and both querycomment and query methods.

Reply:
Done.